output/style.css

```diff
@@ -626,29 +686,311 @@ nav > i:hover {
   background-position: center;
   display: flex;
   flex-direction: column;
+  margin-top: 80px;
 }
 
-.contact-me {
-  /* Primary Text */
+.contact-container {
+  padding: 24px;
+  color: #fff;
+}
 
-  position: absolute;
-  width: 801px;
+.contact-me {
   height: 56px;
-  left: -213px;
-  top: 4734px;
 
   /* Header / 2 */
   font-family: Poppins, sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-
-  /* or 130% */
-  display: flex;
-  align-items: center;
   text-align: center;
 
   /* Neutrals/N0 */
   color: #fff;
 }
+
+.contact-desc {
+  /* Primary Text */
+  height: 102px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin-top: -30px;
```

Comment:
- [OPTIONAL] The left top curve on the blue background for the contact-me section is missing at breakpoints 767px. Kindly look into it and fix.

![your-form-area](https://user-images.githubusercontent.com/26597432/215273877-24616887-fa68-4418-bf5b-8382ac6880ea.png)